CMakeLists.txt

```diff
@@ -66,8 +66,17 @@ message(STATUS "R_LDFLAGS        = ${R_LDFLAGS}")
 message(STATUS "R_LIBRARY_BASE   = ${R_LIBRARY_BASE}")
 message(STATUS "R_LIBRARY_BLAS   = ${R_LIBRARY_BLAS}")
 message(STATUS "R_LIBRARY_LAPACK = ${R_LIBRARY_LAPACK}")
-message(STATUS "R_VERSION_MAJOR = ${R_VERSION_MAJOR}")
-message(STATUS "R_VERSION_MINOR = ${R_VERSION_MINOR}")
+message(STATUS "R_VERSION_MAJOR  = ${R_VERSION_MAJOR}")
+message(STATUS "R_VERSION_MINOR  = ${R_VERSION_MINOR}")
+
+# Install R package hera
+# ======================
+if(EMSCRIPTEN)
+# 
+else()
+    execute_process(COMMAND ${R_COMMAND} CMD INSTALL ../hera)
```

Comment:
This might not be the final solution, but that gets us the current version of hera installed for now.

Reply:
perhaps it belongs more in the `make` then the `cmake` ?